com.unity.mobile.notifications/Runtime/Unified/QueryLastRespondedNotificationOp.cs

```diff
@@ -0,0 +1,114 @@
+using UnityEngine;
+#if UNITY_ANDROID
+using Unity.Notifications.Android;
+#endif
+
+namespace Unity.Notifications
+{
+    /// <summary>
+    /// The state of the query for last responded notification.
+    /// Returned by <see cref="QueryLastRespondedNotificationOp.State"/>
+    /// </summary>
+    public enum QueryLastRespondedNotificationState
+    {
+        /// <summary>
+        /// Operation is ongoing, wait for next frame and check again.
+        /// </summary>
+        Pending,
+
+        /// <summary>
+        /// Operation is complete, app was launched normally, no notification was tapped.
+        /// </summary>
+        NoRespondedNotification,
```

Comment:
I assume this means "Not responded to notification" ? a bit hard to decipher the value meaning